install/README.md

```diff
@@ -127,8 +129,67 @@ Server version:  0.3.439
 ```
 
 **Note:** If the server version is "?" then the `fn` CLI cannot reach the Fn server.  
-If this happens it's likely you have something else running on port 8080 or you 
-started the server on a different port but forgot to set the `FN_API_URL`. 
+If this happens it's likely you have something else running on port 8080 or you
+started the server on a different port but forgot to set the `FN_API_URL`.
+
+
+## Fn Manual Install
+The steps to install Fn manually are described in this section. The system requirements are the same as those outlined for the script installation.
+
+### Download Fn CLI
+Download the CLI for your operating system. For this example, files are saved to the `~/Downloads` directory.
+
+1. Open the Fn project release directory in your browser: <https://github.com/fnproject/cli/releases/>
+    * You should see a list of executables for supported operating systems.
+2. Click on the Fn executable for your operating system.
+3. Save the file locally.
+
+Sample executables:
+
+| Operating System | Executable |
+| ------------- |:-------------:|
+| MacOS | fn_mac |
+| Linux | fn_linux |
+| Alpine Linux | fn_alpine |
+| Windows | fn.exe |
+
+### Copy/Install the Executable
+Now make the Fn file executable and available on your machine.
+
+
+#### For MacOS Systems
+* Open a Terminal Window.
+* Copy the file to executable directory with the correct name.
+
+>**Note:** If you use homebrew, the following commands should work fine. If you do not, you may need to precede the commands with `sudo`.
+
+>```sh
+> mv ~/Downloads/fn_mac /usr/local/bin/fn
```

Comment:
My assumption is that the install script failed because the user didn't have write access to restricted folders like /usr/local/bin and so would have to install manually in a local folder like ${HOME}/bin and make sure it's in their $PATH.  This would apply to all platforms.